pkg/azuredisk/controllerserver.go

```diff
@@ -839,6 +845,35 @@ func (d *Driver) getSnapshotByID(ctx context.Context, resourceGroup, snapshotID,
 	return generateCSISnapshot(sourceVolumeID, &snapshot)
 }
 
+// GetSourceDiskSize returns: disk size, error
+func (d *Driver) GetSourceDiskSize(resourceGroup, diskName string) (*int32, error) {
```

Comment:
what about merge `GetSourceDiskSize` and `GetSourceDiskSizeHelper` functions as one and write a ut? thanks.